src/index.html

```diff
@@ -11,7 +11,185 @@
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>HTML Form</h1>
+    <form>
+      <fieldset class="persInfo">
+
+        <legend>Personal information</legend>
+        Surname:
+        <input
+        type="text"
+        name="lastname"
+        autocomplete="off"
+        required
+        >
+        <br>
+
+        Name:
+        <input
+        class="input"
+        type="text"
+        name="firstname"
+        autocomplete="off"
+        required
+        >
```

Comment:
you need to use a label tag for each input so that every input could be activated by clicking on the corresponding label. it should look like:
```suggestion
        <label for="firstname">
          Name:
          <input
            class="input"
            type="text"
            name="firstname"
            autocomplete="off"
            required
          >
        </label>
```

Reply:
check other places please